buildrunner/validation/config.py

```diff
@@ -110,18 +115,19 @@ def validate_push(
                         name = f"{name}:latest"
 
                 if isinstance(push, StepPushCommitDict):
-                    names = [f"{push.repository}:{tag}" for tag in push.tags]
+                    if not push.tags:
+                        name = f"{push.repository}:latest"
```

Comment:
Latest is not the default. The only default is the specific build commit.

Reply:
Thanks for catching that! I was thinking of how docker defaults to latest if nothing is specified. Fixed that.